math/mathcore/CMakeLists.txt

```diff
@@ -177,6 +176,7 @@ ROOT_STANDARD_LIBRARY_PACKAGE(MathCore
     src/TStatistic.cxx
     src/UnBinData.cxx
     src/triangle.c
+    inc/VectorizedTMath.hxx
```

Comment:
This is not the right fix. You cannot list headers here, only source files, as you can see from above list.